src/components/editor/InputLookupQA.jsx

```diff
@@ -6,197 +6,195 @@ import Swagger from 'swagger-client'
 import { connect } from 'react-redux'
 import { getProperty } from '../../reducers/index'
 import { changeSelections } from '../../actions/index'
+import { defaultValuesFromPropertyTemplate } from '../../Utilities'
 
-const AsyncTypeahead = asyncContainer( Typeahead )
+const AsyncTypeahead = asyncContainer(Typeahead)
 
 class InputLookupQA extends Component {
-    constructor( props ) {
```

Comment:
This component, for some reason, was using 4-space indentation instead of 2-space indentation. That is making this diff look a bit larger than it actually is. Suggest toggling "No Whitespace" button in GitHub UI.